config/environment.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
 module.exports = function(/* environment, appConfig */) {
-  return { };
+  return {
+    emberResponsiveDefaults: {
```

Comment:
I was just looking through a few environment files in my apps and it looks like most addons follow one of two patterns with naming here:

1. Drop the "ember" prefix - so here it would just be "responsive"
2. Use the addon name - so here it would be "emberResponsive"

Just a thought - I don't feel strongly either way.